src/modules/referral/services/queries.ts

```diff
@@ -117,7 +117,8 @@ export const updateStickyReferralAddressesForDepositor = () => {
             order by d2."depositDate" desc
             limit 1
             ) d3 on d1."depositorAddr" = d3."depositorAddr"
-        where d1."depositorAddr" = $1
+        where d1."depositorAddr" = $1 
+          and d1."depositDate" >= $2
```

Comment:
When a deposit `d` is processed by the DepositReferralConsumer to compute the sticky referral address, update the `stickyReferralAddress` only for the deposits with a `depositDate` >= `d`